multinode_integration_tests/tests/ip_change_test.rs

```diff
@@ -54,5 +53,5 @@ fn receiving_announce_from_router_produces_ipchange_gossip() {
     // Have the mock router announce a change in public IP.
     // Verify that the mock Node receives IpChange Gossip.
     // Connect a client and send a request. Verify that the request shows up at the mock Node.
-    todo! ("Finish me")
+    todo!("Finish me")
```

Comment:
This `todo!()` seems serious. Do you have a plan for it? Shouldn't it be driven away yet in this card?

Reply:
Yup, I have a plan for it. It's not done  yet, but the plan is there.